src/Augurk/Controllers/V2/AnalysisController.cs

```diff
@@ -48,24 +48,16 @@ public AnalysisController()
         /// <param name="groupName">Name of the group that the feature belongs to.</param>
         [Route("reports")]
         [HttpPost]
-        public async Task<HttpResponseMessage> PostAnalysisReport(AnalysisReport analysisReport, string productName, string version)
+        public async Task<ActionResult> PostAnalysisReport([FromBody]AnalysisReport analysisReport, string productName, string version)
```

Comment:
These ActionResult<T> should make the Swagger documentation better since it now knows what is being returned. I haven't verified this yet, since the Swagger stuff doesn't work right now, but it should be an improvement.